MyProject/cls/Table.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace cls
+{
+    public class Table
+    {
+        public int TblNumber { get; private set; }
+        public int Guests { get; private set; }
+
+        //_order[100] - question regarding whether to use array or collection?
```

Comment:
for now go with Array, if you would like to use something else, go with HashSet<T>, read a documentation, understand what are the complexity of adding/getting items to/from this collection, what are the specific things about this sort of collection, and if you are fully with it, use it within your ptoject.